src/components/Payroll/PayrollHistory/PayrollHistoryPresentation.tsx

```diff
@@ -116,7 +108,7 @@ export const PayrollHistoryPresentation = ({
         <div className={styles.timeFilterContainer}>
           <Select
             value={selectedTimeFilter}
-            onChange={onTimeFilterChange}
+            onChange={onTimeFilterChange as (value: string) => void}
```

Comment:
Nit: why do we need to cast the value here?

Reply:
Fixsed, we need to cast it as a Time Option type 😄